netflix/src/components/lottie/Landing.tsx

```diff
@@ -0,0 +1,23 @@
+'use client';
+
+import Lottie from 'react-lottie-player';
+import netflixAnimation from './netflixAnimation.json';
+import { useRouter } from 'next/navigation';
+import FooterLine from '../common/FooterLine';
+
+export default function Landing() {
+  const router = useRouter();
+
+  return (
+    <div>
+      <Lottie
+        animationData={netflixAnimation}
+        style={{ display: 'flex', width: '100vw', height: '80vh', paddingTop: '8rem' }}
```

Comment:
이부분도 tailwindcss로 충분히 구현가능한 부분이예요!

```suggestion
className="flex w-screen h-[80vh] pt-32"
```